rasa_core/events/__init__.py

```diff
@@ -1038,8 +1044,57 @@ def as_story_string(self):
     def _from_parameters(cls, parameters):
         return AllEventsReset(parameters.get("evts"),
                         parameters.get("timestamp"))
-            
+
+    def as_dict(self):
+        d = super(AllEventsReset, self).as_dict()
+        d.update({"evts": self.evts})
+        return d
+
     def apply_to(self, tracker):
         evts = deserialise_events(self.evts)
         for e in evts:
             tracker.events.append(e)
+
+
+class DeleteUpdateEvents(Event):
+    """Action to reset tracker including tracker events.
+    Then update tracker events."""
+
+    type_name = "delete_update_events"
+
+    def __init__(self, evts, timestamp=None):
+        self.evts = evts
+        super(DeleteUpdateEvents, self).__init__(timestamp)
+
+    def __hash__(self):
+        return hash(32143124314)
```

Comment:
this hash is always set to this value? Is it used?

Reply:
Not really